book/src/configuration.md

```diff
@@ -18,6 +18,7 @@ Here is an example that defines two custom syntaxes:
 ```toml
 [general]
 default_syntax = "foo"
+automatic_trim_block = true
```

Comment:
I think there should be some explanation of the behavior of this option in the text below.

Reply:
Agreed!